### ISSUE TITLE
Type error Future<List<BluetoothDevice>>  when importing flutter_ftms

### ISSUE THREAD

**Issue by basvdijk** [Malte2036/flutter_ftms#17]
When I use `import 'package:flutter_ftms/flutter_ftms.dart';` not call any function, and run `flutter build ios` I get:

```
Error (Xcode): ../../../.pub-cache/hosted/pub.dev/flutter_ftms-1.1.2/lib/src/bluetooth.dart:25:12: Error: A value of
type 'Future<List<BluetoothDevice>> Function(List<Guid>)' can't be returned from an async function with return type
'Future<List<BluetoothDevice>>'.
```

`flutter clean` followed by `flutter pub get` did not help.

Any ideas on what could cause this?

## Steps to reproduce
- Create new project `flutter create .`
- Add flutter_ftms `flutter pub add flutter_ftms`
- Add import line `import 'package:flutter_ftms/flutter_ftms.dart';` on top of `main.dart`
- Run: `flutter build ios`

**Comment by Malte2036** [comment]
Should be fixed in [flutter_ftms: 1.1.3](https://pub.dev/packages/flutter_ftms) ✨

**Comment by basvdijk** [comment]
@Malte2036 yes it builds now! Thanks for your quick fixing!
